### ISSUE TITLE
identiers incorrectly parsed as keywords when they start with the same letters

### ISSUE THREAD

**Issue by luggage66** [interdependence/tree-sitter-htmldjango#4]
```
{% get_foo as foo %}
```

Correctly parses as:

```
template [0, 0] - [1, 0]
  unpaired_statement [0, 0] - [0, 20]
    tag_name [0, 3] - [0, 10]
    keyword [0, 11] - [0, 13]
    variable [0, 14] - [0, 17]
      variable_name [0, 14] - [0, 17]
```

but if the variable has any keyword in the name is fails:

```
{% get_foo as randomfoo %}
```

Which parses as:

```
template [0, 0] - [1, 0]
  unpaired_statement [0, 0] - [0, 26]
    tag_name [0, 3] - [0, 10]
    keyword [0, 11] - [0, 13]
    keyword [0, 14] - [0, 20]  <- "random" keyword
    ERROR [0, 20] - [0, 23]   <- "foo"
```

**Comment by interdependence** [comment]
Thanks for finding and reporting this! I have written a new test and will see if I can fix it. Something to do with whitespace in the attribute rule.

**Comment by interdependence** [comment]
Should be fixed in ea71012.

I changed the keyword rule (and a few others with potential conflicts) to explicitly require a trailing whitespace. I also removed the standalone whitespace handling rule because it isn't needed.

**Comment by luggage66** [comment]
Working great, thanks!
